docs/IOS_PROJ_INTEGRATION.md

```diff
@@ -14,8 +14,22 @@ Explaining all of cocoapods is not within the scope of this document, however a
 
 ## Cocoapods gradle Integration
 
-Starting with 1.3.30, Kotlin has provided a gradle plugin which allows the Kotlin Native library to be referenced as a Cocoapods dependency (As of [#76](https://github.com/touchlab/KaMPKit/pull/76), KampKit uses a Touchlab fork of this plugin to expand it's capabilities). The integration adds a gradle task that generates a `podspec` that includes everything needed to be referenced by cocoapods. We are using the available cocoapods code block to customize our podspec, which is located in the `shared/build.gradle`.
+Starting with 1.3.30, Kotlin has provided a gradle plugin which allows the Kotlin Native library to be referenced as a Cocoapods dependency (As of [#76](https://github.com/touchlab/KaMPKit/pull/76), KampKit uses a Touchlab fork of this plugin to expand it's capabilities). The integration adds a gradle task that generates a `podspec` that includes everything needed to be referenced by cocoapods. We are using an extension of the available cocoapods code block to customize our podspec, which is located in the `shared/build.gradle`.
 
+```
+cocoapodsext {
+    summary = "Common library for the KaMP starter kit"
+    homepage = "https://github.com/touchlab/KaMPKit"
+    framework {
+        isStatic = false
+        export(Deps.kermit)
+        transitiveExport = true
+    }
+}
+```
+Note that you need to apply the `co.touchlab.native.cocoapods` plugin for the forked plugin. 
+
+The non-forked code block can be seen below.
```

Comment:
I don't think the code below is from the non-forked version as it doesn't have the isStatic version. but the lack of "ext" is confusing. I'd suggest just removing it

Reply:
I removed that second section.